pf/tfgen/not_supported.go

```diff
@@ -150,7 +150,6 @@ func (u *notSupportedUtil) schema(path string, schema *tfbridge.SchemaInfo) {
 	u.assertIsZero(path+".Elem", schema.Elem)
 	u.fields(path, schema.Fields)
 	u.assertIsZero(path+".Asset", schema.Asset)
-	u.assertIsZero(path+".Default", schema.Default)
```

Comment:
This is slightly complicated because there are three places where defaults apply:

Check - this is for resources
Invoke - this is for calling functions
CheckConfig - this is for the provider-level "global" configuration itself

Shall we tackle all of these in the same PR? If not we can move the warnings around so that it wants only for Invoke and CheckConfig.

Reply:
Actually CheckConfig has some pain I'm working through now so perhaps best left out of this PR.

Reply:
Understand your concerns, but how to only remove the warning for `CheckConfig` what I'm currently try to implement?

Reply:
Sorry for the late reply here, I didn't have time to write sample code; basically you'd need to keep track which structure you're traversing; if resources then remove the warning; if functions or provider-level config then keep it.